Formula/a/agda.rb

```diff
@@ -2,20 +2,19 @@ class Agda < Formula
   desc "Dependently typed functional programming language"
   homepage "https://wiki.portal.chalmers.se/agda/"
   license "BSD-3-Clause"
-  revision 3
 
   stable do
-    url "https://hackage.haskell.org/package/Agda-2.6.4.1/Agda-2.6.4.1.tar.gz"
-    sha256 "23248a9b3c50c81ea4751518a66f2a6144617b6a5a9202686b54e33bc9cbd080"
+    url "https://hackage.haskell.org/package/Agda-2.6.4.1/Agda-2.6.4.2.tar.gz"
```

Comment:
It looks like upstream deprecated this version (https://hackage.haskell.org/package/Agda-2.6.4.2)?

Looks like some regressions were found. May want to wait for 2.6.4.3 - https://github.com/agda/agda/pull/7154

Reply:
I'll keep this open and update it for 2.6.4.3 when that releases, then!